egs/wsj/s5/steps/segmentation/convert_rttm_to_utt2spk_and_segments.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
I searched for an invocation of this script in your PR and didn't find it.  Maybe not necessary?